README.md

```diff
@@ -157,6 +158,10 @@ There are multiple supported backends, which can be selected at runtime in the e
 | `/gpu/hip/shared`          | Optimized pure HIP kernels using shared memory    | Yes                   |
 | `/gpu/hip/gen`             | Optimized pure HIP kernels using code generation  | No                    |
 ||
+| **SYCL Native**            |
+| `/gpu/sycl/ref`            | Reference pure SYCL kernels                       | Yes                   |
+| `/gpu/sycl/shared`         | Optimized pure SYCL kernels using shared memory   | Yes                   |
```

Comment:
Are there also CPU impls? `ceed-backend-list.h` says yes?

Reply:
Oh, I guess there are? I haven't heard anything about them. They're also not built when building with SYCL.